item.rb

```diff
@@ -19,10 +21,10 @@ def can_be_archived?
     current_date - Date.parse(@publish_date).year > 10
   end
 
-  #    def label=(label)
-  #     @label = label
-  #     @label.items.push(self) unless @label.items.inculede?(self)
-  #   end
+  # def label=(label)
+  #  @label = label
+  #  @label.items.push(self) unless @label.items.inculede?(self)
+  # end
```

Comment:
Hi @AtillaTahak. You made a spelling error at the level of line 26. I think the method you wanted to use is `include` not `inclulede`. Could you please correct the spelling?